_data/sites.json

```
@@ -17357,6 +17377,17 @@
         ]
     },
 
+    {
+        "name": "VCRDB / Valorant Crosshair Database",
+        "url": "https://www.vcrdb.net/profile",
+        "difficulty": "medium",
+        "notes": "Before deleting your account, make sure you have deleted all Crosshairs you have created, otherwise account deletion will fail with an unknown error.",
+        "domains": [
+            "vcrdb.net",
+            "www.vcrdb.net"
```

```suggestion
            "vcrdb.net"
```
Second entry was redundant.